src/scss/foundations/base.scss

```diff
@@ -0,0 +1,82 @@
+@use "typography";
+@use "spacings";
+
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+}
+
+html {
+  @include typography.globalSettings();
+  box-sizing: border-box;
+  overflow-x: hidden;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-size: inherit;
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+strong {
+  @include typography.getStrongWeight();
+}
+
+img {
+  display: block;
+  width: 100%;
+}
+
+svg {
+  display: inline-block;
+  fill: currentColor;
+  height: auto;
+  stroke: currentColor;
+  width: auto;
+}
+
+input[type="search"]::-webkit-search-decoration,
+input[type="search"]::-webkit-search-cancel-button,
+input[type="search"]::-webkit-search-results-button,
+input[type="search"]::-webkit-search-results-decoration,
+input[type="password"]::-ms-reveal,
+input[type="password"]::-ms-clear {
+  display: none;
+}
+
+button,
+input,
+optgroup,
+select,
+textarea,
+input[type="submit"],
+fieldset {
+  appearance: none;
+  background: none;
+  border: none;
+  margin: 0;
+  padding: 0;
+
+  &:focus {
+    outline: none;
+  }
+}
```

Comment:
Magari questi li possiamo mettere in `foundations/base/form.scss` per avere un minimo di separazione del contesto.